cloc-1.64.pl

```diff
@@ -537,7 +537,7 @@ BEGIN
    "lang_no_ext|lang-no-ext=s"               => \$opt_lang_no_ext         ,
    "yaml"                                    => \$opt_yaml                ,
    "csv"                                     => \$opt_csv                 ,
-   "csv_delimeter|csv-delimiter=s"           => \$opt_csv_delimiter       ,
+   "csv_delimiter|csv-delimiter=s"           => \$opt_csv_delimiter       ,
```

Comment:
This is a bit odd, there are two divergences between the first and second thing, the `_`/`-` and the `e`/`i`